### ISSUE TITLE
Isochrones with Uber traffic data 

### ISSUE THREAD

**Issue by redfrexx** [GIScience/openrouteservice#1500]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature/Enhancement Proposal

The current prototypical Uber traffic data integration in the `implement_uber_traffic` branch only works for routing, but not for isochrones. Adding support for isochrones would enable us to do several high impact analyses and visuals (isochrones across day) which we could show at upcoming conferences. For that to work, it would be necessary to have it until mid August. Not necessary to have it in `master`.

**Comment by rabidllama** [comment]
The problem isn't with the isochrones themselves, but a missing timezone decleration. If you set that in the AbstractTrafficGraphStorage (currently isn't initialised) then you can get isochrones. However, that is then hardcoding it to a single time zone which isn't the proper solution

**Comment by MichaelsJP** [comment]
You're right. We were just serving Germany so far. If you're not hardcoding it, you deal with this monstrosity :smile:  --> 
![image](https://github.com/GIScience/openrouteservice/assets/22392104/1e5d4191-7114-471f-aee7-e662f19b59c5)

**Comment by MichaelsJP** [comment]
We had a half working solution before, where a library was used to translate coordinates into Time Zones. But it was utterly slow and not actively maintained. We had to remove it. So far, we haven't found a suitable solution. Maybe it's worth just providing a parameter to select the time zone manually for now. It should be sufficient for the smaller Uber traffic data use cases. What do you think?

**Comment by MichaelsJP** [comment]
It could be worth looking into using https://github.com/evansiroky/timezone-boundary-builder. Kind of the successor to http://efele.net/maps/tz/world/

**Comment by MichaelsJP** [comment]
Just found this issue: https://github.com/GIScience/openrouteservice/issues/1317

**Comment by aoles** [comment]
Uber traffic data is not available anymore. Instead, please have a look at HERE traffic data.
